src/Areas/BicepSchema/Commands/BicepSchemaGetCommand.cs

```diff
@@ -53,19 +53,24 @@ public override Task<CommandResponse> ExecuteAsync(CommandContext context, Parse
                 {
                     return Task.FromResult(context.Response);
                 }
-                var bicepSchemaService = context.GetService<IBicepSchemaService>() ?? throw new InvalidOperationException("Bicep schema service is not available.");
-                var resourceTypeDefinitions = bicepSchemaService.GetResourceTypeDefinitions(
-                    s_serviceProvider.Value,
-                    options.ResourceType!);
 
                 TypesDefinitionResult result = SchemaGenerator.GetResourceTypeDefinitions(s_serviceProvider.Value, options.ResourceType!);
                 List<ComplexType> response = SchemaGenerator.GetResponse(result);
 
-                context.Response.Results = response is not null ?
-                    ResponseResult.Create(
+                if (result is not null)
```

Comment:
Previously, `context.Response.Results` should be null or not depends on if `response` is null or not. Now it depends on `result`. I wonder if it is possible that result is not null, but response is null.

Reply:
Good point. The old branching logic was checking response so there is no reason to change that.